api-reference/10 UI Components/GridBase/1 Configuration/columnChooser/search/editorOptions.md

```diff
@@ -0,0 +1,213 @@
+
+See the [TextBox Configuration](/Documentation/ApiReference/UI_Components/dxTextBox/Configuration/) for properties you can specify in this object.
+
+#include widgets-config-object-option-note with {
+    optionName: "editorOptions"
+}
+
+---
+##### jQuery
+
+    <!--JavaScript-->
+    $(function(){
+        $("#{widgetName}").dx{WidgetName}({
+            // ...
+            columnChooser: {
+                // ...
+                search: {
+                    editorOptions: { 
+                        placeholder: 'Search column',
+                        mode: 'text',
+                        onOptionChanged: (e) => {
+                            if(e.name === 'onValueChanged') {
+                                // handle the option change here
+                            }
+                        }
+                    },
+                    // ...    
+                },
+            },
+        })
+    });
+
+##### Angular
+
+    <!-- tab: app.component.html -->
+    <dx-{widget-name} ... >
+        <dxo-column-chooser ... >
+            <dxo-search
+                [editorOptions]="searchEditorOptions"
+                // ...
+            ></dxo-search>
+        </dxo-column-chooser>
+    </dx-{widget-name}>
+
+    <!-- tab: app.component.ts -->
+    import { Component } from '@angular/core'; 
+
+    @Component({ 
+        selector: 'app-root', 
+        templateUrl: './app.component.html', 
+        styleUrls: ['./app.component.css'] 
+    }) 
+
+    export class AppComponent { 
+        searchEditorOptions;
```

Comment:
you need to set type for this field. Use type TextBoxConfiguration if possible, if not object
```suggestion
        searchEditorOptions: TextBoxConfiguration;
```